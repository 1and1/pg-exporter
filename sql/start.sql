show port \gset
\setenv PGPORT :port
show unix_socket_directories \gset
\setenv PGHOST :unix_socket_directories
select current_database() as database \gset
\setenv PGDATABASE :database

\! ./bin/pg_exporter --log.level=warn & echo $! > pid; sleep 0.1

\! curl -sf localhost:9135/metrics | grep postgresql_up
