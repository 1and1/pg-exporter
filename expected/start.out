show port \gset
\setenv PGPORT :port
show unix_socket_directories \gset
\setenv PGHOST :unix_socket_directories
select current_database() as database \gset
\setenv PGDATABASE :database
\! ./bin/pg_exporter --log.level=warn & echo $! > pid; sleep 0.1
\! curl -sf localhost:9135/metrics | grep postgresql_up
# HELP postgresql_up Whether the PostgreSQL server is up.
# TYPE postgresql_up gauge
postgresql_up 1
